### PR TITLE
add scheduler jobs

### PR DESCRIPTION
**Pull request by chongr**

### CODE REVIEW

File: modules/queue/main.tf [hunk 1]
```diff
@@ -301,3 +301,14 @@ resource "google_monitoring_alert_policy" "dataflow_freshness" {
...
+resource "google_cloud_scheduler_job" "job" {
+  for_each = {for job_config in var.taskhawk-scheduler_jobs:  job_config.name => job_config}
+  name        = each.key
+  description = each.value.description
+  schedule    = each.value.schedule
+
+  pubsub_target {
+    topic_name = google_pubsub_topic.topic.id
+  }
+}
```
Comment (maroux) [inline modules/queue/main.tf:314]: missing newline.

File: modules/queue/variables.tf [hunk 2]
```diff
@@ -81,3 +81,17 @@ variable "dataflow_freshness_alert_notification_channels" {
   type        = list(string)
   default     = []
 }
+
+variable "taskhawk_scheduler_job_config" {
+type = object({
```
Comment (maroux) [inline modules/queue/variables.tf:86]: is this terraform 0.13 syntax? Because we're still on 0.12..

File: modules/queue/main.tf [hunk 3]
```diff
@@ -301,3 +301,14 @@ resource "google_monitoring_alert_policy" "dataflow_freshness" {
 
   notification_channels = var.dataflow_freshness_alert_notification_channels
 }
+
+resource "google_cloud_scheduler_job" "job" {
```
Comment (maroux) [inline modules/queue/main.tf:305]: probably needs some permissions as well? I'm not sure..

File: modules/queue/variables.tf (hunk 2 above)
Comment (chongr) [reply modules/queue/variables.tf:86]: Assuming this guy is trustworthy I think it is .12 https://github.com/rberlind/terraform-0.12-examples#rich-value-types

Comment (maroux) [reply modules/queue/variables.tf:86]: niceee TIL!

**Review by yosh — APPROVED** [review]

**Review by maroux — APPROVED** [review]

**Pull request merged**